UndertaleModTool/SampleScripts/FindAndReplace.csx

```diff
@@ -25,8 +25,8 @@ if (!ScriptQuestion("This will make changes across all of the code! Are you sure
 bool case_sensitive = ScriptQuestion("Case sensitive?"); 
 bool multiline = ScriptQuestion("Multi-line search?"); 
 bool isRegex = ScriptQuestion("Is regex search?");
-String keyword = SimpleTextInput("Enter search terms", "Search box below", "", multiline);
-String replacement = SimpleTextInput("Enter replacement term", "Search box below", "", multiline);
+String keyword = SimpleTextInput("Enter search terms", "Search box below", "", multiline).Replace('\v', '\n');
+String replacement = SimpleTextInput("Enter replacement term", "Search box below", "", multiline).Replace('\v', '\n');
```

Comment:
This case should get handled in SimpleTextInput rather than in the script itself.